### PR TITLE
Update README with project setup instructions

### PR DESCRIPTION
**Pull request by ww2497**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,30 @@ The public facing website for BrickHack.
 
 # Getting Started
```
Comment (peterkos) [inline README.md:9]: If you want to go for maximum Cool README I'd suggest putting a screenshot of the preregister site right at the top! See the [brickhack-mobile-ios readme](https://github.com/codeRIT/brickhack-mobile-ios/) for how I centered that image if you'd like.

File: README.md [hunk 2]
```diff
@@ -8,16 +8,30 @@ The public facing website for BrickHack.
...
 
-#### Code environment
+## Code environment
```
Comment (peterkos) [inline README.md:11]: Title not needed imo, description is clear as-is

File: README.md [hunk 3]
```diff
@@ -8,16 +8,30 @@ The public facing website for BrickHack.
...
+$ npm install
+```
+
+### Running the application
+
+```
+$ npm start
+```
+
+You should then be able to access the site at `localhost:3000`.
```
Comment (peterkos) [inline README.md:34]: Good instructions!

**Review by peterkos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ww2497) [reply README.md:9]: I feel like that might be a little less relevant for this since it's a website you can easily visit.

**Pull request merged**